docs/addresses.md

```diff
@@ -18,6 +19,10 @@ description: List of Standard Contract Addresses
 [wrose-verify-mainnet]: https://sourcify.dev/#/lookup/0x8Bc2B030b299964eEfb5e1e0b36991352E56D2D3
 [wrose-verify-testnet]: https://sourcify.dev/#/lookup/0xB759a0fbc1dA517aF257D5Cf039aB4D86dFB3b94
 
+[message-executor]: https://im-docs.celer.network/developer/development-guide/message-executor
+[message-executor-source]: https://github.com/celer-network/im-executor
+[message-executor-testnet]: https://explorer.oasis.io/testnet/sapphire/address/0x9C850D230FFFaCEf1E2D1741a00080856630e455
+
 # Celer cBridge Tokens (Mainnet)
```

Comment:
Please use consistent chapter names, e.g.
- Celer cBridge Tokens (Mainnet)
- Celer cBridge Tokens (Testnet)

This fix should probably go here https://github.com/oasisprotocol/sapphire-paratime/blob/main/docs/_fetch-cbridge-tokens.py#L30?

Reply:
> Please use consistent chapter names, e.g.

👍 

> This fix should probably go here https://github.com/oasisprotocol/sapphire-paratime/blob/main/docs/_fetch-cbridge-tokens.py#L30?

I agree with your previous [comment](https://github.com/oasisprotocol/sapphire-paratime/pull/299#discussion_r1549277903) that these aren't tokens, so will suggest another section for them.

Separately, the list of tokens are out of date, and I'll patch that.

Reply:
> There are multiple mainnet executors, one for each project, but we could monitor the messagebus contract address
For testnet it's wise to monitor both executor and the messagebus contract address - as people may run their own executor

from @CedarMist